test_plans/header-file.asciidoc

```diff
@@ -0,0 +1,44 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for header files and namespaces
+
+This is a test plan for the header files and namespaces as described in SYCL 2020 Section 4.3.
+
+Estimated development time is one day.
+
+== Testing scope
+
+=== Backend coverage
+
+All the tests described below are performed for any SYCL backend.
+
+=== Device coverage
+
+All the tests described below are performed only on the default device that is selected on the CTS command line.
+
+== Tests
+
+The following combinations for namespaces and header files exist:
+
+|===
+|    |`::sycl` |`::CL::sycl`
+
+|`sycl/sycl.hpp`
+|1
+|2
+
+|`CL/sycl.hpp`
+|3
+|4
+|===
+
+Configurations 1 and 4 should compile, and 2 and 3 should not. Two versions are proposed:
```

Comment:
I think the spec does not require combination 2 or 3 to fail.  It only provides a guarantee that combinations 1 and 4 are valid.  Therefore, I do not think we should test that 2 or 3 fail.

Reply:
Right, I have dropped the requirement.